### DISCUSSION TITLE
Extract Strain Rate from PorousFlowVolumetricStrain

### DISCUSSION THREAD

**Discussion by jrpatterson79** [idaholab/moose#26030] [Q&A Modules: Porous Flow] [answered]
Hello Moose community,

I'm actively developing and de-bugging a multi-dimensional HM model. The general concept is a 2D horizontal fracture plane embedded in a 3D volume using a conforming mesh generated with Gmsh, both represented as separate subdomains.

My understanding is that [PorousFlowVolumetricStrain](https://mooseframework.inl.gov/source/materials/PorousFlowVolumetricStrain.html) computes the principal and volumetric strain rates. Is it possible to have this material output the zz component of the strain rate at each time step / is there a minimal working example I can follow as a reference? Or is there a different / better approach I should be considering to output strain rates?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Use the `output_properties` parameter to trigger a material property output
If you need to find the name of the material properties first, you can get them from show_material_properties in the Debug block

Guillaume

**Reply by jrpatterson79** [reply]
Thank you @GiudGiud
